spec/TillSpec.js

```diff
@@ -0,0 +1,93 @@
+describe('Till', function() {
+  var till;
+
+  describe('gives the price of', function() {
+
+    it('a Cafe Latte as 4.75', function(){
+      till = new Till();
+      expect(till.retrievePrice('Cafe Latte')).toEqual(4.75);
+    });
+
+    it('a Blueberry Muffin as 4.05', function(){
+      till = new Till();
+      expect(till.retrievePrice('Blueberry Muffin')).toEqual(4.05);
+    });
+  });
+
+  describe('calculates the total price of', function() {
+    it('a Cafe Latte and a Blueberry Muffin', function() {
+      till = new Till();
+      till.addItem('Cafe Latte');
+      till.addItem('Blueberry Muffin')
+      expect(till.calculateSubTotal()).toEqual(8.80);
+    });
+
+    it('two Cafe Lattes', function() {
+      till = new Till();
+      till.addItem('Cafe Latte');
+      till.addItem('Cafe Latte');
+      expect(till.calculateSubTotal()).toEqual(9.50);
+    });
+  });
+
+  describe('calculates tax as', function() {
+    it('£8.64 on an order totalling £100', function() {
+      till = new Till();
+      expect(till.calculateTax(100.00)).toEqual(8.64);
```

Comment:
Maybe `calculateTax()` shouldn't rely on being passed an arbitrary amount, but calculate tax on the the subtotal of items you have in your till